src/org/opensolaris/opengrok/web/PageConfig.java

```diff
@@ -975,7 +975,7 @@ public String getResourcePath() {
     public boolean resourceNotAvailable() {
         getIgnoredNames();
         return getResourcePath().equals("/") || ignoredNames.ignore(getPath())
-                || ignoredNames.ignore(resourceFile.getParentFile().getName())
```

Comment:
What this change does is that it converts the basename check to full path check. The basename is checked in the Filter class anyway so this expands the checking. The whole logic of resourceNotAvailable() is not very clear to me. @tulinkry can you explain the intention here ?